source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -413,6 +413,14 @@ message TransformationAccessChain {
 
   // Adds an access chain instruction based on a given pointer and indices.
 
+  // When accessing a struct, the corresponding indices must be 32-bit integer constants.
+  // For any other composite, the indices can be any 32-bit integer, and the transformation
+  // adds two instructions for each such index to clamp it to the bound, as follows:
+  //
+  // %t1 = OpULessThanEqual %bool %index %bound_minus_one
+  // %t2 = OpSelect %int_type %t1 %index %bound_minus_one
+
```

Comment:
Remove extraneous blank line (one blank line after comment for the transformation and before comment for the first field).

Reply:
Done